cassandra-diagnostics-commons/src/main/java/io/smartcat/cassandra/diagnostics/info/NodeInfo.java

```diff
@@ -14,29 +14,39 @@
      * Information about status of the thrift (active or not).
      */
     public final boolean thriftActive;
+
     /**
      * Information about status of the native transport (active or not).
      */
     public final boolean nativeTransportActive;
+
     /**
      * Node uptime in seconds.
      */
     public final long uptimeInSeconds;
 
     /**
+     * Count of uncaught exceptions.
```

Comment:
I don't have any ideas how to cause some kind of uncaught exception to test if this will be incremented. Any ideas?